SemanticMaps/SemanticMaps.hooks.php

```diff
@@ -82,7 +86,8 @@ public static function addGeoCoordsDefaultFormat( &$format, array $printRequests
 				$hasCoords = false;
 
 				// Loop through the print requests to determine their types.
-				foreach( $printRequests as /* SMWPrintRequest */ $printRequest ) {
+				foreach ( $printRequests as /* SMWPrintRequest */
+						  $printRequest ) {
```

Comment:
Eww alignment

Reply:
wtf... guess the IDE got confused by the weird inline comment. Thanks for pointing this out, I'll remove the comment so it does not get confused.